hack/lib/build/release.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+# This library holds utility functions for building releases.
+
+# os::build::release::check_for_rpms checks that an RPM release has been built
+function os::build::release::check_for_rpms() {
+	if [[ ! -d "${OS_LOCAL_RPMPATH}" || ! -s "${OS_LOCAL_RELEASEPATH}/CHECKSUM" ]]; then
```

Comment:
@stevekuznetsov now I think the `! -s "${OS_LOCAL_RELEASEPATH}/CHECKSUM` is unecessary here since the release binaries are not being built as part of building the RPMs and the images have no use for the release binaries anyway.